### PR TITLE
offzip 0.3.6a (new formula)

### PR DESCRIPTION
**Pull request by tevino**
Should I open standalone Github repo for it so that it has a homepage and repo of its own?

### CODE REVIEW

**Comment by tdsmith** [comment]
No, we'd rather point to whatever the author maintains. Can you please contact the author and ask for a versioned download URL – something like `http://aluigi.altervista.org/mytoolz/offzip-0.3.6a.zip`? Otherwise the formula will break when a new version is uploaded. Thanks!

File: Library/Formula/offzip.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Offzip < Formula
+  version "0.3.6a"
+  homepage "http://aluigi.altervista.org/mytoolz.htm#offzip"
+  url "http://aluigi.altervista.org/mytoolz/offzip.zip"
+  sha256 "71d578d4bcfad52857ef7d2db0b16935ad88aaef493c9b4367d310069efd892b"
+
+  def install
+    system ENV.cc, "offzip.c", "-o", "offzip", "-lz"
+    bin.install "offzip"
```
Comment (mikemcquaid) [inline Library/Formula/offzip.rb]: Is there a `Makefile`?

File: Library/Formula/offzip.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
+  test do
+    system "offzip|grep #{version}"
```
Comment (mikemcquaid) [inline Library/Formula/offzip.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by tevino** [comment]
@tdsmith I contacted the author, but unfortunately he can't help, he simply overrides the same file every time.

File: Library/Formula/offzip.rb (hunk 1 above)
Comment (tevino) [inline Library/Formula/offzip.rb]: I hoped so, but no.

File: Library/Formula/offzip.rb (hunk 2 above)
Comment (tevino) [inline Library/Formula/offzip.rb]: The url contains no version number, so the test is just to ensure the right version is used.
I guess I should just remove it.

**Comment by mikemcquaid** [comment]
I don't think we can include that if that's what the author does, unfortunately. Also, is the `a` on the name signifying an alpha version?

**Comment by tevino** [comment]
@mikemcquaid That's alright, I'm not sure what the `a` means. I guest this should be closed, thanks so far.

**Pull request closed without merging**